badge_server/test_badges.py

```diff
@@ -250,6 +250,13 @@
     status=compatibility_store.Status.SUCCESS,
     timestamp=datetime.datetime(2019, 5, 7, 0, 0, 0))
 
+APACHE_BEAM_GOOGLE_API_CORE_RECENT_PAIR_INCOMPATIBLE_2 = compatibility_store.CompatibilityResult(
+    [package.Package('apache-beam[gcp]'),
```

Comment:
Aren't there messages for these?

Reply:
What do you mean by messages?

Reply:
Like the message that explains the error? I guess we aren't setting those anywhere else because they are only relevant for the details page.

Reply:
OK, you can leave it in this PR I guess, but you should add them for the details tests.